### PR TITLE
DOC: correct min_count param docstring

### PR DESCRIPTION
**Pull request by ZhuBaohe**
correct min_count param docstring for function  ’Series.prod‘， ’Series.sum',  'DataFrame.prod', 'DataFrame.sum'

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -7867,8 +7867,8 @@ def _doc_parms(cls):
 
     .. versionadded :: 0.22.0
 
-       Added with the default being 1. This means the sum or product
-       of an all-NA or empty series is ``NaN``.
+       Added with the default being 0. This means the sum or product
```
Comment (jreback) [inline pandas/core/generic.py]: can you reverse the logic of the final part here. Also can you check the consistency with other text for min_count (in this file)

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/core/generic.py (hunk 1 above)
Comment (ZhuBaohe) [reply pandas/core/generic.py]: I reverse the logic in new commit. If I understand wrong, please give me specific comments. A total of 9 functions share parameter descriptions (Series.sum, Series.prod, Series.product ,  DataFrame.sum, DataFrame.prod, DataFrame.product ,  Panel.sum, Panel.prod, Panel.product).

Comment (TomAugspurger) [reply pandas/core/generic.py]: Do you find it clearer to phrase the second sentence as

    This means the sum of an all-NA or empty Series is 0, and the product of an all-NA or empty Series is 1.

Comment (ZhuBaohe) [reply pandas/core/generic.py]: 😄  Already made the change

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

**Pull request merged**